src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -512,6 +513,174 @@ pass::EliminateConcat::EliminateConcat() {
     this->register_matcher(m, callback);
 }
 
+pass::EliminateConcatSplit::EliminateConcatSplit() {
+    MATCHER_SCOPE(EliminateConcatSplit);
+    auto pattern_concat = pattern::wrap_type<ov::op::v0::Concat>();
+    matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        const auto& pattern_map = m.get_pattern_map();
+        const auto concat = ov::as_type_ptr<ov::op::v0::Concat>(pattern_map.at(pattern_concat));
+        if (!concat || concat->is_dynamic() || concat->get_users().size() == 1) {
+            return false;
+        }
+        const auto concat_users = concat->get_users();
+        auto concat_inputs = concat->inputs();
+        auto concat_axis = concat->get_axis();
+        if (concat_axis < 0)
+            concat_axis = concat_axis + concat->get_output_shape(0).size();
+
+        bool need_opt = false;
+        std::vector<std::tuple<std::shared_ptr<Node>, int64_t, int64_t>> slice_out_index_in_cocat;
+        for (const auto& user : concat_users) {
+            if (ov::is_type<ov::op::v1::StridedSlice>(user)) {
+                auto strided_slice_node = ov::as_type_ptr<ov::op::v1::StridedSlice>(user);
+                if (!strided_slice_node) {
+                    return false;
```

Comment:
Does this mean that all Concat operation consumers should be StridedSlice and these StridedSlice should split the tensor into the parts and these parts be equal to the original parts before Concat? 
if so, please add it to the transformation description

Reply:
@itikhono Yes, u are right! Will update the description.